CMakeLists.txt

```diff
@@ -328,7 +328,7 @@ if(MIOPEN_USE_MLIR)
     # REQUIRED is not supported before cmake 3.18
     find_library(LIBMLIRMIOPEN MLIRMIOpen REQUIRED)
     if(NOT BUILD_SHARED_LIBS AND MIOPEN_USE_COMGR)
-        message(FATAL_ERROR "Potential symbol conflict between mlir and comgr in static build")
+        message(FATAL_ERROR "Potential symbol conflict between mlir and comgr in static build, please configure with -DMIOPEN_USE_MLIR=Off")
```

Comment:
I would revert this. In some cases the correct recommendation might be switching off COMGR (not MLIR).

Reply:
Sounds good